### PR TITLE
parse title in JSONParser for LBLD

### PR DESCRIPTION
**Pull request by daemon1024**
Fixes https://github.com/publiclab/leaflet-blurred-location-display/pull/136#issuecomment-901314290 https://github.com/publiclab/plots2/pull/10050#issuecomment-901273164

### CODE REVIEW

**Comment by daemon1024** [comment]
image

It's working 🥳

**Review by jywarren — APPROVED** [review]

**Comment by jywarren** [comment]
Hooray! OK - publishing!

**Comment by daemon1024** [comment]
It's published https://www.npmjs.com/package/leaflet-environmental-layers/v/2.4.8 :D

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "leaflet-environmental-layers",
-  "version": "2.4.7",
+  "version": "2.4.8",
```
Comment (jywarren) [inline package.json:3]: Version bump here and in corresponding package-lock.json!

Comment (daemon1024) [reply package.json:3]: Yup, I missed the bump in 2.4.7 for the lock file at https://github.com/publiclab/leaflet-environmental-layers/commit/702ebe1877ad9d816bd9856798607b325b7ffe13 😅

**Pull request merged**